kpaamcamcommands/analyze.py

```diff
@@ -0,0 +1,135 @@
+"""
+
+"""
+import pathlib
+import itertools
+
+import pandas
+from lingpy import LexStat, Alignments, csv2list
+from lingpy.convert.plot import plot_heatmap
+from lingpy.evaluate.acd import bcubes
+
+from lexibank_kpaamcam import Dataset
+
+
+def register(parser):
+    pass
+
+
+def run(args):
+    ds = Dataset()
+    cldf = ds.cldf_reader()
```

Comment:
Since the command lives in the same code base as the data, we have easy access to the CLDF data.